mandatory/2-function-creation.js

```diff
@@ -6,7 +6,12 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (typeof num === "number" && num % 2 === 0 && num <= 100){ 
+    return true;
+  }else return false ;
```

Comment:
slight optimisation, if you're `return`ing from the `if` above, you don't actually need the `else` here ; e.g. 
```
function validate(num) {
   if (...) {
      return true
   }
   return false
}
```

As a further optimisation you could make, the statement in the `if` statement returns a `true`/`false` anyway (e.g. `typeof num === number && num < 10` will evaluate to `true` or `false`. Seeing as that's what you're returning anyway, you can `return` that statement e.g.
```
function validate(num) {
  return typeof num === "number" && num % 2 === 0 && num <= 100
}
```